### DISCUSSION TITLE
pygada vs pydov

### DISCUSSION THREAD

**Discussion by pjhaest** [DOV-Vlaanderen/pygada#9] [General]
Hi @GuillaumeVandekerckhove,  
pleased to see that you are proceeding with the data of DOV in the Python environment!  Did you consider adding the missing exploration functionalities to existing pacakages, and the missing functionalities for data extraction to pydov? This could reduce your workload in the future if either pydov data or modules of processing packages change.  
Anyway, looking forward to your developments!

**Comment by GuillaumeVandekerckhove** [comment]
Dear @pjhaest, 
I had the pleasure that @Roel reviewed and adapted my initial coding (by f.e. creating the new functionalities).
He can tell you more if this is going to be incorporated in pydov.
Thank you for your interest and feedback!
